src/target/adiv5.h

```diff
@@ -404,6 +404,8 @@ const void *adiv5_pack_data(uint32_t dest, const void *src, uint32_t *data, alig
 void firmware_mem_write_sized(adiv5_access_port_s *ap, uint32_t dest, const void *src, size_t len, align_e align);
 void firmware_mem_read(adiv5_access_port_s *ap, void *dest, uint32_t src, size_t len);
 void firmware_ap_write(adiv5_access_port_s *ap, uint16_t addr, uint32_t value);
+void mm32l0_firmware_mem_write_sized(
```

Comment:
Don't forget to remove this when you put the mm32-specific write function into stm32f1.c